data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1082 @@
+			`Your communicator beeps and you realize you have received a message from Remington. There's a Captain of one of their security detachments waiting to meet you in the spaceport. Apparently there've been developments while you were in flight. Now might be the last opportunity to sort anything else out for a little while.`
+				accept
+
+
+# Ensure there are some impressive fleets at each system.
+mission "Hai Reveal [B01-A] The Blockade Goes Up"
+	landing
+	invisible
+	source "Mountaintop"
+	to offer
+		has "Hai Reveal [A14] A Time of Trouble: offered"
+	to complete
+		never
+	npc
+		government "Republic"
+		system "Rigel"
+		personality heroic opportunistic staying
+		fleet
+			names "republic capital"
+			fighters "syndicate fighter"
+			variant
+				"Carrier (Mark II)" 2
+				"Cruiser (Mark II)" 6
+				"Combat Drone" 36
+				"Lance (Gatling)" 8
+				"Frigate (Mark II)" 12
+				"Rainmaker (Mark II)" 8
+				"Rainmaker (Mark II Old Weapons)" 8
+	npc
+		government "Republic"
+		system "Nihal"
+		personality heroic opportunistic staying
+		fleet
+			names "republic capital"
+			fighters "syndicate fighter"
+			variant
+				"Carrier (Mark II)" 2
+				"Cruiser (Mark II)" 6
+				"Combat Drone" 36
+				"Lance (Gatling)" 8
+				"Frigate (Mark II)" 12
+				"Rainmaker (Mark II)" 8
+				"Rainmaker (Mark II Old Weapons)" 8
+	npc
+		government "Syndicate"
+		system "Mirfak"
+		personality heroic staying
+		fleet "Large Syndicate" 8
+	npc
+		government "Syndicate"
+		system "Zaurak"
+		personality heroic staying
+		fleet "Large Syndicate" 8
+
+
+# The fact that you have to actively go to the spaceport to proc this mission is intentional. It allows the player to take a pause to reorganize themselves if they need to, B01 stays active as a reminder.
+# The fight in this mission is very large, but it's also a complete roflstomp. The pirates are outnumbered 2-1, and out-gunned significantly worse than that.
+# You as the player have one job: make sure everyone jumps in together, and don't die. During the first campaign it was really important the fights were all balanced for you to be able to tip the balance.
+# THIS mission is designed to give you a spectacle. A little visual demonstration of what a real battle could potentially look and feel like with low stakes for you. It's a visual feast, not a challenge.
+mission "Hai Reveal [B02] Join the Fleet"
+	name "Join the Fleet"
+	description "Join the fleet in <waypoint> and move on Stormhold."
+	source "Sunracer"
+	waypoint "Sumar"
```

Comment:
Since the NPCs always travel to the player, forcing a "rendezvous point" doesn't make much sense. The player could just wait for the ships to come to them and never have to go to Sumar.

Reply:
The narrative purpose is so that the Hai fleet doesn't fly into places it's not meant to.

Reply:
Well that can't be enforced regardless, so I'd simply remove the waypoint.

Reply:
No, but with the waypoint, if the player takes the fleet elsewhere they know that they're consciously departing from the narrative.

Reply:
Just toss it.